nemo/collections/nlp/data/datasets/joint_intent_slot_dataset/joint_intent_slot_dataset.py

```diff
@@ -157,6 +157,7 @@ def __init__(
         pad_label=128,
         ignore_extra_tokens=False,
         ignore_start_end=False,
+        do_lower_case=False,
```

Comment:
why do we need `do_lower_case` arg? if bert-base-uncased is used, it will do_lower by default

Reply:
It is not going to be always bert_uncased. That is why I am adding to be able use other bert models.

Reply:
It is not going to be always bert_uncased. That is why I am adding to be able use other bert models.